### ISSUE TITLE
Mobilito Selenium test sometimes fail

### ISSUE THREAD

**Issue by Shriukan33** [transport-nantes/tn_web#1063]
While working on an unrelated app (photo), I noticed that a mobilito test failed, but not all the time.
It seems to have to do with selenium sometimes having time to load certain elements, so it might be vulnerable to CPU's performance.

**Comment by Shriukan33** [comment]
After a rebase, I couldn't reproduce the error even after a dozen of tests launching.
I'll put this aside until it becomes a thing again.

**Comment by JeffAbrahamson** [comment]
Timing errors are very fragile.  When you get a repro, hold it dearly to your chest and dig in.  It won't stay around for long.  Until you understand what's caused the bug, you don't have much to say.

Above all, don't let yourself believe you dodged a bullet.  You should document extensively what you learn and summarise that documentation in detail in the ticket that tracks the issue.

**Comment by Shriukan33** [comment]
From memory :

* Sometimes, mobilito/tests.py's test_flag_session test would fail, stating that the `dropdownMenuLink` would not be available for a click.
* This would happen about 50 to 66% of the time, at the time I discovered the issue.
* I tried to introduce action chains and used `webdriver.until`, although it did seem to get a little better, it didn't fix the problem entirely. 
* Adding a time.sleep(2) made the problem disappear (cf #1064) at the time. 
* The next week, on master without fixes, after a little more than a dozen attempts, the test just won't fail.
